MPF.md

```diff
@@ -14,6 +14,7 @@ various aspects of the system.
 * `services/main.cf` - Contains an empty bundle agent main where custom policy
   can be integrated.
 * `services/autorun/` - Automatically included policy files.
+* `.no-distrib/` - A directory that is excluded from policy updates.
```

Comment:
to be more clear here possibly, "excluded from policy updates from agents." maybe? Just to clarify "policy updates" means an agent running cf-agent -KIf update.cf.

Reply:
ack